superset-frontend/src/CRUD/CollectionTable.tsx

```diff
@@ -181,15 +205,75 @@ export default class CRUDCollection extends React.PureComponent<
     }));
   }
 
+  sortColumn(col: string, sort = 0) {
```

Comment:
Shouldn't `sort` be a `boolean` indicating if it's ascending or descending? Do we have more than two possible states?